src/main/java/browserview/BrowserComponent.java

```diff
@@ -123,12 +130,25 @@ private ChromeDriverEx createChromeDriver() {
         ChromeDriverEx driver = new ChromeDriverEx(options, isTestChromeDriver);
         WebDriver.Options manage = driver.manage();
         if (!isTestChromeDriver) {
+            setWindowSize(manage);
+            initialiseJNA();
+        }
+        return driver;
+    }
+
+    public void setWindowSize(WebDriver.Options manage) {
```

Comment:
This should probably have a comment. It positions the browser based on the main window, right? The UI is like an implicit parameter.